content/post/activity_hours.md

```diff
@@ -57,24 +45,25 @@ It normally takes less than 5 minutes to get the cluster running and fully opera
 
 The next step is to download the datasets. data.world has a size limit which prevents from uploading files bigger than 50MB, so we decided to upload ours to Google Drive for ease of use.
 
-<pre class="verbatim">
+```
```

Comment:
![screenshot_2017-07-24_09 37 22](https://user-images.githubusercontent.com/1312023/28512816-bb42cc56-7053-11e7-9d28-933c8e4cde3c.png)

Nothing explodes without it